src/components/Text/BoldText.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { colors, typography } from "../../theme";
+import constants from "../../theme/constants";
+import StyledText from "./StyledText";
+
+const BoldStyledText = StyledText.extend`
```

Comment:
This is not just bold text - it's bold text and upper case ... I would recommend removing uppercase property